tests/localstack/integration/test_close_investigation.py

```
@@ -0,0 +1,333 @@
+    # Verify status changed to INACTIVE
+    describe_response = ecs_client.describe_task_definition(taskDefinition=task_def_arn)
+    assert describe_response['taskDefinition']['status'] == 'INACTIVE'
+
+    # Deregister again (idempotency check - should not fail)
+    try:
+        deregister_response = ecs_client.deregister_task_definition(taskDefinition=task_def_arn)
+        # Some implementations return INACTIVE, others may succeed silently
+        if 'taskDefinition' in deregister_response:
+            assert deregister_response['taskDefinition']['status'] == 'INACTIVE'
+        print("✓ DeregisterTaskDefinition is idempotent (no error on second call)")
+    except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)